recipes/kfac-jax/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "kfac-jax" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kfac-jax-{{ version }}.tar.gz
+  sha256: cf308272aee8148109121af96b2f99221fce40aac5a7f1f5287a0f433a21a1fc
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - absl-py >=0.12.0
+    - immutabledict >=2.2.1
+    - numpy >=1.21
+    - distrax >=0.1.3
+    - jax >=0.4.7
+    - jaxlib >=0.4.7
+    - dm-tree >=0.1.7
+    - optax >=0.1.4
+    - typing-extensions >=4.2.0
+
+test:
+  imports:
+    - docs
```

Comment:
There is an additional docs directory installed here which should not be included